src/auth/auth.controller.ts

```diff
@@ -0,0 +1,83 @@
+import { Body, Controller, Get, HttpCode, HttpStatus, Param, Patch, Post, Res, UseGuards } from '@nestjs/common';
+import { Response } from 'express';
+import { ChangePasswordDto, LoginUserDto } from './dto';
+import { AuthService } from './auth.service';
+import {
+  ApiResponse,
+  ConfirmResponse,
+  RecoveryPasswordResponse,
+  Tokens,
+  UpdateResponse,
+  UserDataResponse,
+} from '@Types';
+import { GetToken, GetUserData, GetUserId, Public } from '../common/decorators';
+import { MtGuard, RtGuard } from '../common/guards';
+import { RecoveryPasswordDto } from './dto/recovery-password.dto';
+import { ConfirmDto } from './dto/confirm.dto';
+
+@Controller('auth')
+export class AuthController {
+  constructor(private authService: AuthService) {
+  }
+
+  @Public()
```

Comment:
Ok. Czyli jednak jest domyślnie zabezpieczone wszystko, a @Public odpowiada za to, że ściezka jest widoczna z neta?
Still, nie jest weryfikowana rola z tego co rozumiem. Czyli każdy User może wykonać akcje np. dla Admina.